dockerfiles/Debian10.dockerfile

```diff
@@ -0,0 +1,78 @@
+# vim: ft=dockerfile
+
+FROM debian:10
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# TERM=screen is fairly neutral and works with xterm for example, for others
+# you might need to pass -e TERM=<terminal>, like rxvt-unicode.
+ENV TERM screen
+ENV OSCODENAME buster
+
+# Set debbuild/debbuild version
+ARG DBVERSION=20.01.0
+
+# Install all packages required
+# hadolint ignore=DL3008,DL3015
+RUN apt-get update -qq && \
+    apt-get install -qqy \
+    wget \
+    pycodestyle \
+    pyflakes3 \
+    python-django-common \
+    python3-cheetah  \
+    python3-coverage \
+    python3-distro \
+    python3-distutils \
+    python3-django \
+    python3-dns \
+    python3-dnsq  \
+    python3-future \
+    python3-ldap3 \
+    python3-netaddr \
+    python3-pip \
+    python3-pycodestyle \
+    python3-pytest \
+    python3-setuptools \
+    python3-simplejson  \
+    python3-sphinx \
+    python3-tornado \
+    python3-tz \
+    python3-yaml \
+    liblocale-gettext-perl \
+    lsb-release \
+    xz-utils \
+    bzip2 \
+    dpkg-dev \
+    tftpd-hpa \
+    createrepo \
+    rsync \
+    xorriso\
+    fakeroot \
+    patch \
+    pax \
+    git \
+    apache2 \
+    libapache2-mod-wsgi-py3 \
+    systemd && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install debbuild packages (not in any repository)
+WORKDIR /var/cache/apt/archives
+RUN wget https://github.com/debbuild/debbuild/releases/download/$DBVERSION/debbuild_$DBVERSION-0ubuntu18.04_all.deb && \
+    wget https://github.com/debbuild/debbuild-macros/releases/download/0.0.2/debbuild-macros_0.0.2-0ubuntu18.04_all.deb  && \
+    dpkg -i debbuild_$DBVERSION-0ubuntu18.04_all.deb && \
+    dpkg -i debbuild-macros_0.0.2-0ubuntu18.04_all.deb && \
+    rm -f \*.deb
```

Comment:
Wouldn't this repository count? http://download.opensuse.org/repositories/Debian:/debbuild/Debian_10/

Reply:
It does. Only learned from its existence yesterday.
I have update the DL location to the packages and mentioned it.
Setting up the repo with release keys etc. is too much of a hassle (for now) and will only increase the container image.